cdap-app-fabric/src/main/java/co/cask/cdap/app/store/Store.java

```diff
@@ -486,4 +487,15 @@ void setWorkflowProgramStart(Id.Program programId, String programRunId, String w
    * @return the {@link WorkflowToken} for the specified workflow run
    */
   WorkflowToken getWorkflowToken(Id.Workflow workflowId, String workflowRunId);
+
+  /**
+   * Used by {@link co.cask.cdap.gateway.handlers.WorkflowStatsSlaHttpHandler} to get the list of all the completed
+   * workflow runs within a time range
+   * @param programId
+   * @param startTime
+   * @param endTime
+   * @return the list of {@link co.cask.cdap.internal.app.store.WorkflowDataset.WorkflowRunRecord} for a given workflow
+   */
+  List<WorkflowDataset.WorkflowRunRecord> getWorkflowRuns(final Id.Program programId,
```

Comment:
should use ``Id.Workflow`` instead of ``Id.Program`` right? as this is specific to workflow.